src/Network/Network/FirewallPolicy/ManagedRules/ManagedRuleSet/ManagedRuleGroupOverride/ManagedRuleOverride/AzureApplicationGatewayFirewallPolicyManagedRuleOverride.cs

```diff
@@ -40,6 +40,13 @@ public class AzureApplicationGatewayFirewallPolicyManagedRuleOverride : NetworkB
         [ValidateNotNullOrEmpty]
         public string Action { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Describes the override sensitivity to be applied when rule matches.")]
+        [ValidateSet("None", "Low", "Medium", "High", IgnoreCase = true)]
+        [ValidateNotNullOrEmpty]
```

Comment:
Why can this not be null?  Infact for all ruleset other than DDoS ruleset, this will be null.

Reply:
it is used for check when the field is present, if it's not specified, then the check wont be used